src/widget/wstarrating.h

```diff
@@ -32,7 +32,9 @@ class WStarRating : public QWidget, public WBaseWidget {
     void mouseMoveEvent(QMouseEvent *event);
     void mouseReleaseEvent(QMouseEvent *event);
     void leaveEvent(QEvent *);
-    
+    virtual bool event(QEvent* pEvent);
+    void fillDebugTooltip(QStringList* debug);
```

Comment:
It is done. I added "virtual" and  "/override/" as comment even if I've not been able to find this string somewhere else in the sources. Please, tell me if I'm wrong.